src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -43,61 +42,53 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     [TestClass]
     public class BindingProcessImplTests
     {
-        private ConfigurableServiceProvider serviceProvider;
-        private Mock<ISonarQubeService> sonarQubeServiceMock;
-        private ConfigurableHost host;
-        private TestLogger logger;
-
-        [TestInitialize]
-        public void TestInitialize()
```

Comment:
The tests don't need the `host` any more so I took the opportunity to get rid of the `TestInitialize`.